beacon_node/client/src/notifier.rs

```diff
@@ -313,6 +314,7 @@ pub fn spawn_notifier<T: BeaconChainTypes>(
 
             eth1_logging(&beacon_chain, &log);
             merge_readiness_logging(current_slot, &beacon_chain, &log).await;
+            capella_readiness_logging(current_slot, &beacon_chain, &log).await;
```

Comment:
I think we only want to disable logging of the execution node being offline when `beacon_chain.is_time_to_prepare_for_capella()` is `true`. Otherwise on testnets where the `CAPELLA_FORK_EPOCH` is shortly after `BELLATRIX_FORK_EPOCH`, it will not log when the execution endpoint is misconfigured for the merge.

Reply:
I've made the change - lmk what you think.